### PR TITLE
add govuk styling

### PR DESCRIPTION
**Pull request by dvoong**
The / route should now give you something like this,

image

feel free to edit the `templates/base.html`, `templates/index.html` and `static/css/custom.js`, @marcuspp I believe your PR is for some of this stuff. These templates are here more for examples and testing the styles

### CODE REVIEW

File: .dockerignore [hunk 1]
```diff
@@ -0,0 +1 @@
+node_modules/*
```
Comment (marcuspp) [inline .dockerignore:1]: Is this still needed?

File: Dockerfile [hunk 2]
```diff
@@ -1,6 +1,9 @@
 FROM python:3.6
 
 RUN apt-get update -y
+RUN curl -sL https://deb.nodesource.com/setup_14.x | bash -
```
Comment (marcuspp) [inline Dockerfile:4]: Update `scripts/install_node.sh` and call that

File: scripts/compile_assets.sh [hunk 3]
```diff
@@ -2,5 +2,6 @@
 
 source ./scripts/functions.sh
 
+echo `more package.json`
```
Comment (marcuspp) [inline scripts/compile_assets.sh:5]: Remove debug

File: templates/data_store_service/base.html [hunk 4]
```diff
@@ -0,0 +1,132 @@
...
+
+        <div class="govuk-width-container">
+            <div class="govuk-phase-banner">
+                <p class="govuk-phase-banner__content">
+		    <strong class="govuk-tag govuk-phase-banner__content__tag ">
+			Alpha
+		    </strong>
+                    <span class="govuk-phase-banner__text">
+			This is a new service – your&nbsp;<a class="govuk-link" href="">feedback</a>&nbsp;will help us to improve it.
+		    </span>
```
Comment (marcuspp) [inline templates/data_store_service/base.html:68]: mixed tabbing. Sometimes 4 sometimes 2.

File: .dockerignore (hunk 1 above)
Comment (dvoong) [reply .dockerignore:1]: yeah I think it's fine

**Pull request merged**